.buildbot.sh

```diff
@@ -70,8 +70,12 @@ for mode in "" "--release"; do
     cargo build ${mode} -p ykcapi;
 done
 
-cargo test
-cargo test --release
+# We test lots of times to check for non-deterministic failures.
+for i in `seq 50`; do \
+    cargo test \
+    cargo test --release \
+; done
```

Comment:
This `; ` isn't necessary if you remove the `\` from the previous line I think?

Reply:
Yeah, for some reason I wrote a make target in a shell script :P my bad.

Fixed in 0f1c9ba01b51d5c8ba3ec3825c12525be0b528e5